burrow/src/main.rs

```diff
@@ -27,17 +31,23 @@ enum Commands {
 #[derive(Args)]
 struct StartArgs {}
 
+#[instrument]
 async fn try_main() -> Result<()> {
     burrow::ensureroot::ensure_root();
 
     let iface = TunInterface::new()?;
-    println!("{:?}", iface.name());
+    tracing::info!(interface_name = ?iface.name());
 
     Ok(())
 }
 
 #[tokio::main(flavor = "current_thread")]
 async fn main() {
+    LogTracer::init().expect("first thing in main - a logger shouldn't have been set yet");
+
+    let logger = init_logger_layer().with_subscriber(FmtSubscriber::new());
+    tracing::subscriber::set_global_default(logger).expect("Logger shouldn't be set already");
```

Comment:
Is there a way to configure tracing to print to stdout? perhaps with an environment variable?

Reply:
I'm not sure, but AFAIK you need to add a logger for it to work. The FmtSubscriber logger should respect the RUST_LOG env variable